e2e-tests/support/tasks.js

```diff
@@ -9,6 +9,12 @@ module.exports = {
     const getUsersCollection = () => db.getCollection(usersCollectionName, connectionOptions);
 
     return {
+      log(message) {
+        // eslint-disable-next-line no-console
+        console.log(message);
```

Comment:
Please remove `console.log` Eslint rules will only allow `info, warn and error`.